cvmfs/statistics_database.cc

```diff
@@ -121,7 +121,7 @@ StatisticsDatabase::~StatisticsDatabase() {
 }
 
 
-Stats StatisticsDatabase::GetStats(swissknife::Command *command) {
+Stats StatisticsDatabase::GetStats(const swissknife::Command *command) {
```

Comment:
This method doesn't need to take a `swissknife::Command` pointer, just a `Statistics` one.
No need to couple the `StatisticsDatabase` class and the `swissknife::Command` classes.